index.html

```diff
@@ -18,7 +18,11 @@ <h2 class="subhead">New Activity</h2>
           <section class="category-timer-wrapper">
             <article class="timer-box-wrapper hidden">
               <h3 class="timer-box-header">Deep Breathing</h3>
-              <h1 class="countdown-clock">05:00</h1>
+              <div class="countdown-clock">
+                <h1 class="countdown-minutes">00</h1>
+                <h1>:</h1>
+                <h1 class="countdown-seconds">00</h1>
+              </div>
```

Comment:
will need to adjust styling as a result of this change; Steve put in on our list on the project board